symphony-bdk-http/symphony-bdk-http-api/src/main/java/com/symphony/bdk/http/api/tracing/MDCUtils.java

```diff
@@ -0,0 +1,65 @@
+package com.symphony.bdk.http.api.tracing;
+
+import lombok.RequiredArgsConstructor;
+import org.apiguardian.api.API;
+import org.slf4j.MDC;
+
+import java.util.Map;
+
+/**
+ * Helper class for managing {@link MDC} in multi-thread applications.
+ */
+@API(status = API.Status.STABLE)
+public final class MDCUtils {
```

Comment:
MdcUtils ?

Reply:
http://www.slf4j.org/api/org/slf4j/MDC.html in uppercase... I wanted to follow the naming